templates/library.html

```diff
@@ -51,8 +51,13 @@ <h1 style="margin-top: 00px;" id="libraryName">{{library.name}}</h1>
       <div data-library-version="{{version}}" class="assets library-table-container">
           <table style="margin-top: 10px;" cellpadding="0" cellspacing="0" border="0" class="table table-striped" id="example">
             <tbody>
+              <tr id="control-hidden" {{^hasHidden}}style="display: none"{{/hasHidden}}>
+                <td colspan="2">
+                    <a id="show-hidden" style="cursor: pointer">Click to show all files</a>
```

Comment:
I wonder if we can use wording with more information, like:
`Some files are hidden, click to show all files`